### PR TITLE
Update server.template.conf. Fixes #276

### LINKED ISSUE #276: Warning regarding unknown options while running on Ubuntu 22.04
While using the shared server_template.conf I'm getting the below warnings for `outdir` and `logfile`.

```
arjun@hp-envy:~/CM/repos/local/cache/b722357835384985/power-dev/ptd_client_server$ power_server -c server.template.conf 
ptd-server 2022-11-29 13:39:09,400 [WARNING] server.template.conf: ignoring unknown options in section 'server': outdir
ptd-server 2022-11-29 13:39:09,400 [WARNING] server.template.conf: ignoring unknown options in section 'ptd': logfile
ptd-server 2022-11-29 13:39:09,522 [INFO] NTP:offset = -0.044 s, delay = 0.104 s 
ptd-server 2022-11-29 13:39:09,523 [INFO] Sources: {"sources": {"__init__.py": "da39a3ee5e6b4b0d3255bfef95601890afd80709", "client.py": "33ca4f26368777ac06e01f9567b714a4b8063886", "lib/__init__.py": "da39a3ee5e6b4b0d3255bfef95601890afd80709", "lib/client.py": "4c2b78fb4849a7e5b584ef792d82aaed20b17f57", "lib/common.py": "624d0c0acc7c39aaff3674f0b99d6a09da53d1dc", "lib/external/__init__.py": "da39a3ee5e6b4b0d3255bfef95601890afd80709", "lib/external/ntplib.py": "4da8f970656505a40483206ef2b5d3dd5e81711d", "lib/server.py": "cda0cdfaee9bfa1249c64a7dd4f89b7bf1b279f0", "lib/source_hashes.py": "60a2e02193209e8d392803326208d5466342da18", "lib/summary.py": "aa92f0a3f975eecd44d3c0cd0236342ccc9f941d", "lib/time_sync.py": "3210db56eb0ff0df57bf4293dc4d4b03fffd46f1", "server.py": "c3f90f2f7eeb4db30727556d0c815ebc89b3d28b", "tests/unit/__init__.py": "da39a3ee5e6b4b0d3255bfef95601890afd80709", "tests/unit/test_server.py": "99ae15aef722f2000ee6ed1ae1523637bf1ae42b", "tests/unit/test_source_hashes.py": "00468a2907583c593e6574a1f6b404e4651c221a"}, "modules": {"ptd_client_server.lib.common": "lib/common.py", "ptd_client_server.lib.external.ntplib": "lib/external/ntplib.py", "ptd_client_server.lib.server": "lib/server.py", "ptd_client_server.lib.source_hashes": "lib/source_hashes.py", "ptd_client_server.lib.summary": "lib/summary.py", "ptd_client_server.lib.time_sync": "lib/time_sync.py"}}
ptd-server 2022-11-29 13:39:09,523 [INFO] Ready to accept connections at 0.0.0.0:4950
```

### PR DESCRIPTION
**Pull request by arjunsuresh**
The server.template.conf is made consistent with the server.py and also made same as the configuration given [here](https://github.com/octoml/power-dev/blob/fix3/ptd_client_server/README.md#configuration)

### CODE REVIEW

File: ptd_client_server/server.template.conf [hunk 1]
```diff
@@ -4,22 +4,13 @@
 [server]
 # NTP server to sync with before each measurement.
 # See "NTP" section in the README.md.
-# ntpServer: ntp.example.com
+#ntpServer: ntp.example.com
```
Comment (dmiskovic-NV) [inline ptd_client_server/server.template.conf:7]: why remove space?

File: ptd_client_server/server.template.conf [hunk 2]
```diff
@@ -4,22 +4,13 @@
...
-# The name of this sub-directory consists of date, time, label, and mode (ranging/testing).
-# The loadgen log is fetched from the client if the `--send-logs` option is enabled for the client.
-# The name of the directory is determined by the workload script running on the SUT, e.g. `ssdmobilenet`.
-# The power log, named `spl.txt`, is extracted from the full PTDaemon log (`ptdLogfile`)
-outDir: D:\ptd-logs\
 
 # (Optional) IP address and port that server listen on
 # Defaults to "0.0.0.0 4950" if not set
-# listen: 192.168.1.2 4950
+#listen: 192.168.1.2 4950
```
Comment (dmiskovic-NV) [inline ptd_client_server/server.template.conf:12]: why remove space?

File: ptd_client_server/server.template.conf [hunk 3]
```diff
@@ -28,10 +19,6 @@ outDir: D:\ptd-logs\
 # A path to PTDaemon executable binary.
 ptd: D:\PTD\ptd-windows-x86.exe
 
-# A path to a logfile that PTDaemon produces (`-l` option).
-# Note that in the current implementation this file is considered temporary
-# and may be overwritten.
-logFile: logs_ptdeamon.txt
```
Comment (dmiskovic-NV) [inline ptd_client_server/server.template.conf:34]: this is to accommodate new file needed for Uncertainty check?

File: ptd_client_server/server.template.conf (hunk 1 above)
Comment (arjunsuresh) [reply ptd_client_server/server.template.conf]: That was an unintended edit. Fixed now.

File: ptd_client_server/server.template.conf (hunk 2 above)
Comment (arjunsuresh) [reply ptd_client_server/server.template.conf]: That was an unintended edit. Fixed now.

File: ptd_client_server/server.template.conf (hunk 3 above)
Comment (arjunsuresh) [reply ptd_client_server/server.template.conf:34]: Thank you @dmiskovic-NV for the review. 
This is not related to the new check. This logfile entry is not used even by the current `server.py` script and it is reported as a warning.

**Comment by dmiskovic-NV** [comment]
Looks good to me.
Let's see if anyone else chimes in

**Review by araghun — APPROVED** [review]

**Pull request merged**